.github/ISSUE_TEMPLATE/ask_a_question.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Ask a question
+about: Ask a question for FullStory or the community
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Your Questions**
+A clear and concise description of your question and what you are trying to achieve.
+
+**Context**
+A broader context in which you’re asking the question, please provide any context that might be helpful for the recipient to know? This may include your use case, or a description of an unexpected behavior.
```

Comment:
```suggestion
A broader context in which you’re asking the question. Please provide any context that might be helpful for the recipient to know. This may include your use case or a description of an unexpected behavior.
```

Reply:
fixed 👍